website/content/docs/security/acl/acl-rules.mdx

```diff
@@ -143,28 +194,74 @@ On success, the Policy is returned:
 }
 ```
 
-The created policy can now be specified either by name or by ID when
+The policy can now be specified either by name or by ID when
 [creating a token](https://learn.hashicorp.com/tutorials/consul/access-control-setup-production#create-the-agent-token).
 This will grant the rules provided to the [bearer of that token](/api#authentication).
 
-Below is a breakdown of each rule type.
+## Resource and Rule Reference
 
-#### ACL Resource Rules
+The following table provides an overview of the resources you can use to create ACL rules.
 
-The `acl` resource controls access to ACL operations in the
-[ACL API](/api/acl/acl).
+| Resource                      | Description                                                                                                                                              | Segments |
```

Comment:
We have a similar table in the ACL Overview docs. I think we can probably remove that now and link here.